lib/enkf/ensemble_config.cpp

```diff
@@ -573,6 +587,23 @@ void ensemble_config_init_SUMMARY( ensemble_config_type * ensemble_config , cons
   }
 }
 
+void ensemble_config_init_SUMMARY_full( ensemble_config_type * ensemble_config , char * key, const ecl_sum_type * refcase) {
```

Comment:
Perhaps we could replace this code with a function call to to `ensemble_config_init_SUMMARY_full`.
 https://github.com/equinor/libres/blob/ebebba98e2a51431b4f64b9e4ce3316581979e11/lib/enkf/ensemble_config.cpp#L569

Reply:
We need this function in python too though:
`ecl_sum_select_matching_general_var_list`